Bylaws.mediawiki

```diff
@@ -80,6 +80,33 @@
 
 # The organization reserves the right to limit membership based on the capacity of the space.
 
+== Equipment ==
+
+Hacklab operates under a general principle of "all members are equal owners of the space". This principle applies to the tools and materials inside of hacklab as well. Unless something has been labeled as property of a specific member, please consider the tool yours. But remember that it is not just yours, it belongs to everybody. Pursuant to this explanation, the following rules can be summarized as:
+
+# Don't use tools in hacklab in a way that will prevent other members from using them. In other words: don't hog or break stuff.
+# Don't hurt yourself.
+
+=== What to do before using equipment ===
```

Comment:
this whole section feels more like an onboarding item rather than bylaws appropriate.

Reply:
I'd prefer to keep this. I intended it as a TLDR in case someone makes it to this wiki page but doesn't want to read the whole thing.

I've made a small change in commit 1a6bf0bbb1fa7f1291440b6b145bb3bd6dee45a1bb7ea to make it slightly more TLDR by putting the two points thing right at the top and adding bolding.

I'm ok to remove it if people feel strongly about it though.

Reply:
see other